app/helpers/surveys_helper.rb

```diff
@@ -119,6 +119,14 @@ def question_group_locals(surveys_question_group, index, total, is_results_view:
 
   def question_conditional_string(question)
     return '' if question.nil?
+    unless question.conditionals.nil?
```

Comment:
I think this could be refactored to be more concise with less branching. The idea here is that we want to validate the conditional string, and ensure that the question it refers to actually exists. The best way to do that might be to make a separate method that takes a conditional string and returns `true` or `false` based on whether it references a valid conditional question. With a well-named method, the purpose of this check will also be clearer.

Reply:
Okay